index.js

```diff
@@ -0,0 +1,9 @@
+const newsView = require("./src/newsView");
+const newsApi = require("./src/newsApi");
+const newsModel = require("./src/newsModel");
+
+const api = new newsApi();
+const model = new newsModel();
```

Comment:
I don't think these would be needed since you've used default parameters with dependency injection.